packages/dx-react-scheduler-material-ui/src/templates/all-day-panel/wrapper.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = () => ({
+  wrapper: {
+    position: 'relative',
+  },
+});
+
+export const WrapperBase = ({
```

Comment:
`children` is also a part of `restProps`

```js
const WrapperBase = ({ classes, className, ...restProps }) => (
  <div className={classNames(classes.wrapper, className)} {...restProps} />
);
```

Reply:
I think this way is more clear.

Reply:
This is rather strange.

Reply:
We have used this way in many places already, such as [chart](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-chart/src/templates/layout.tsx#L19), [grid](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid-material-ui/src/templates/layout.jsx#L20), [scheduler](https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-scheduler-material-ui/src/templates/layout.jsx#L6).

Reply:
This way hints us that it is a container for something.